gno.land/pkg/gnoweb/components/layouts/head.html

```diff
@@ -44,5 +44,11 @@
   <!-- wallet integrations -->
   <meta name="gnoconnect:rpc" content="{{ .Remote }}" />
   <meta name="gnoconnect:chainid" content="{{ .ChainId }}" />
+
+  <!-- color Scheme Support -->
+  <meta name="color-scheme" content="light dark" />
+
+  <!-- mobile Theme Color -->
+  <meta name="theme-color" light="#FFFFFF" dark="dark" />
```

Comment:
`light="#FFFFFF" dark="dark"` doesn't exist in the meta `theme-color` and has no effect on the browser UI color as expected. 

```suggestion
<meta name="theme-color" content="#FFFFFF" media="(prefers-color-scheme: light)" />
<meta name="theme-color" content="#000000" media="(prefers-color-scheme: dark)" />
```